content/posts/addons-by-default.md

```diff
@@ -0,0 +1,100 @@
+Read the Docs Addons is the new modular implementation of all the features Read the Docs adds on top of your documentation:
+analytics, flyout menu, notifications and search, among others.
+
+Since Read the Docs started, every time it ran a build,
+a set of extra configuration were automatically added to make the integration easier.
+This manipulation happened at Sphinx's `conf.py` and MkDocs's `mkdocs.yml` configuration files,
+and included the following:
+
+- Adding JS and CSS files.
+- Defining the Read the Docs theme by default.
+- Injecting extra data into Sphinx's `html_context` to make it available from Jinja2 templates.
+- Configurating the LaTeX engine for Japanese and Chinese projects.
+- Setting the canonical URL based on the domain defined in Read the Docs.
+- ... and others.
+
+This manipulation has worked great for **+10 years** for most of our projects.
+However, for some users this "behind the scenes" manipulation was hard to understand
+and mislead what was going on when building on Read the Docs,
+since results may differ from when building locally.
+
+The introduction of Read the Docs Addons [last year](https://blog.readthedocs.com/addons-flyout-menu-beta/),
+has proved that all these features can be added on top of the documentation without having to manipulate project's configuration at build time,
+removing these differences between building locally and on Read the Docs;
+making users to always get the result they were expecting.
+
+We've been enabling Read the Docs Addons by default to all projects using `build.commands` since then
+--which was a workaround for those users wanting to skip this "behind the scenes" config manipulation,
+and it also showed great success.
+
+Another step towards this direction was the removal of [YAML manipulation](https://about.readthedocs.com/blog/2024/03/mkdocs-yaml-manipulation/) for all MkDocs projects, on Aril 2024.
+Since then, all projects building with MkDocs are already using Read the Docs Addons and they haven't reported any big issues.
+
+All of this step-by-step changes have given us the confidence to make the last step and **plan the removal of the config manipulation for Sphinx projects** as well,
+which is the most used documentation tool on Read the Docs,
+and that's why we left it for the end.
+
+
+## Deprecation timeline
+
+We understand this change is going to affect _all the projects_ building on Read the Docs,
+so we have a timeline to make this deprecation easier for our users.
+Keep these dates in mind and prepare your projects for the changes:
+
+- **July 29, 2024**: enable Read the Docs Addons for _all **new** projects_ created.
+- **October 7, 2024**: enable Read the Docs Addons for _**all projects**_.
+
+
+## How does it affect my projects?
+
+It's worth to note there are two big different types of changes that will affect your projects:
+at build time, and visually.
+Depending on the configuration of your project, it could be affected by one, both or none of them.
```

Comment:
```suggestion
There are two big different types of changes that will affect your projects:
at build time, and visually.
Depending on the configuration of your project, it could be affected by one, both or none of them.
```